src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -95,7 +95,7 @@ internal override bool Initialize(ApplicationEngine engine)
             for (int i = 0; i < Blockchain.CommitteeMembersCount; i++)
             {
                 ECPoint pubkey = Blockchain.StandbyCommittee[i];
-                RegisterCandidate(engine.Snapshot, pubkey);
+                RegisterCandidate(engine.Snapshot, pubkey, "Genesis candidate");
```

Comment:
In particular, I like this as well.

But perhaps the whole description could be optional.
If not provided we could just write "Description not provided".
And perhaps it needs an easy way to update.